repos/system_upgrade/el7toel8/actors/persistentnetnames/tests/test_persistentnetnames.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from leapp.libraries.common import persistentnetnames
+from leapp.models import Interface, PCIAddress, PersistentNetNamesFacts
+from leapp.snactor.fixture import current_actor_context
+
+
+def interface_mocked(i=0):
+    return Interface(
+        name='n'+str(i),
```

Comment:
Just out of curiosity compared with the way I would do that and this is the result:
```python

In [2]: %timeit "n{}".format(1)                                                                                         
156 ns ± 0.644 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [3]: %timeit "n" + str(1)                                                                                            
168 ns ± 0.253 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```
** this is not th requirement to change **

Reply:
No, this is a good point. Thanks!